helm/alfresco-infrastructure/values.yaml

```diff
@@ -30,7 +20,7 @@ nginx-ingress:
 alfresco-infrastructure:
   alfresco-identity-service:
     enabled: true
-  alfresco-event-gateway:
```

Comment:
Why are we changing this for a persistence PR?

Reply:
fixed, was a typo.